src/theme/StyledComponents.styled.ts

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components/native';
```

Comment:
Good thing to have some shared StyledComponents, nice job :smile:
I think we can move it to different place, proposal: `src/components/SharedComponents.styled.ts`.